cd2_views/create_account_hierarchy_view.sql

```diff
@@ -1,4 +1,3 @@
-CREATE VIEW account_hierarchy_vw AS 
```

Comment:
I think we should keep this line, so that executing this file will create a new view.

Reply:
or, change to `CREATE OR REPLACE VIEW account_hierarchy_vw` so that re-running will replace the view with any new changes.